notebooks/data-sources/TestGrid/metrics/test_pass_failures.ipynb

```diff
@@ -2,91 +2,143 @@
  "cells": [
```

Comment:
<p>Do you want to limit the data uploaded? Looks like there are 20 Million records, but then only pushing 1 million to ceph. Is that your intention here?</p><p> </p><pre class="ql-syntax" spellcheck="false">cc.upload_to_ceph(combined_df.head(1000000)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes, the data is quite large to be uploaded to ceph so trying to store a smaller size